internal/protoc/downloader.go

```diff
@@ -71,23 +71,26 @@ func newDownloader(config settings.Config, options ...DownloaderOption) (*downlo
 	if downloader.config.Compile.ProtobufVersion == "" {
 		downloader.config.Compile.ProtobufVersion = vars.DefaultProtocVersion
 	}
-	if downloader.protocBinPath != "" || downloader.protocWktPath != "" {
+	if downloader.protocBinPath != "" || downloader.protocWKTPath != "" {
 		if downloader.protocURL != "" {
 			return nil, fmt.Errorf("cannot use protoc-url in combination with either protoc-bin-path or protoc-wkt-path")
 		}
-		if downloader.protocBinPath == "" || downloader.protocWktPath == "" {
+		if downloader.protocBinPath == "" || downloader.protocWKTPath == "" {
 			return nil, fmt.Errorf("both protoc-bin-path and protoc-wkt-path must be set")
 		}
 		cleanBinPath := filepath.Clean(downloader.protocBinPath)
 		if _, err := os.Stat(cleanBinPath); os.IsNotExist(err) {
 			return nil, err
 		}
-		cleanWktPath := filepath.Clean(downloader.protocWktPath)
-		if _, err := os.Stat(cleanWktPath); os.IsNotExist(err) {
+		cleanWKTPath := filepath.Clean(downloader.protocWKTPath)
+		if _, err := os.Stat(cleanWKTPath); os.IsNotExist(err) {
+			return nil, err
+		}
+		if _, err := os.Stat(filepath.Join(cleanWKTPath, "google", "protobuf")); os.IsNotExist(err) {
```

Comment:
It’s really easy to just also call fileInfo.IsDir() on the return value from stat

Reply:
Sure, I can add that in!